provider/spark.go

```diff
@@ -359,8 +359,17 @@ func (e *EMRExecutor) PythonFileURI(store SparkFileStore) (filestore.Filepath, e
 
 // Need the bucket from here
 func (db *DatabricksExecutor) PythonFileURI(store SparkFileStore) (filestore.Filepath, error) {
-	filePath := config.GetSparkRemoteScriptPath()
-	return store.CreateFilePath(filePath)
+	relativePath := config.GetSparkRemoteScriptPath()
+	filePath, err := store.CreateFilePath(relativePath)
+	if err != nil {
+		return nil, fmt.Errorf("could not create file path: %v", err)
+	}
+	if store.FilestoreType() == filestore.S3 {
```

Comment:
Should we set s3 to always use s3a?

Reply:
I think we're going to have to test this with EMR to ensure we can make that change.